app-service-proxy/test/headertest/utils.go

```diff
@@ -116,7 +114,7 @@ func GetXAuthHeader(url, _ string) (string, error) {
 	//preText = responseBody
 	fmt.Println("html link : ", pageTitle)
 	fmt.Println("preText : ", preText)
-	tempDir := os.TempDir() // Uses the default OS temporary directory
+	/*tempDir := os.TempDir() // Uses the default OS temporary directory
```

Comment:
Can we remove these sections that are commented out?

Reply:
done